plugins/linear-genome-view/src/LinearGenomeView/components/TrackLabel.tsx

```diff
@@ -73,7 +73,7 @@ const TrackLabel = React.forwardRef<HTMLDivElement, Props>(function (
     },
     ...(session.getTrackActionMenuItems?.(trackConf) || []),
     ...track.trackMenuItems(),
-  ].sort((a, b) => (b.priority || 0) - (a.priority || 0))
+  ].sort((a, b) => (b?.priority || 0) - (a?.priority || 0))
```

Comment:
nitpick but is it true that this needs a nullish check?

Reply:
might be able to remove

Reply:
It was failing on that in testing -- again i wasn't able to explain why exactly it was happening (might have something to do w nextjs and alignments) but yeah.